tests/UI/specs/UIIntegration_spec.js

```diff
@@ -530,7 +530,7 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
     it('should fail correctly when db information in config is incorrect', function (done) {
         testEnvironment.configOverride = {
             database: {
-                host: '127.50.50.50',
+                host: 'localhost',
```

Comment:
Should be maybe use `phpServer.HTTP_HOST` or `phpServer.REMOTE_ADDR` here? In my case MySQL is listening to `apache.piwik/192.168.33.11`. As the test will fail either way `localhost` should be fine I reckon